securedrop/bin/run

```diff
@@ -9,7 +9,6 @@ cd "${REPOROOT}/securedrop"
 source "${BASH_SOURCE%/*}/dev-deps"
 
 run_redis &
-run_supervisor &
```

Comment:
we'll also need to use this systemd service in the dev env else we'll reopen #4733

Reply:
Good point. I'll take a look.

Reply:
I think running systemd in a Docker container is going to be [more trouble](https://developers.redhat.com/blog/2016/09/13/running-systemd-in-a-non-privileged-container/) [than it's worth](https://developers.redhat.com/blog/2019/04/24/how-to-run-systemd-in-a-container/) just to run `rqworker` and `rqrequeue` in the dev container.